agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1601,7 +1633,8 @@ bool backhaul_manager::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_r
         // Handle the CMDU message. If the message was processed locally
         // (by the Backhaul Manager), this function will return 'true'.
         // Otherwise, it should be forwarded to the slaves.
-        if (handle_1905_1_message(cmdu_rx, src_mac)) {
+        Socket *destinationAP = nullptr;
```

Comment:
We're not camelCasing:
```suggestion
        Socket *dest_slave = nullptr;
```

Also, maybe it's a good idea to update the comment explaining the new variable.
And extract this entire change into a separate commit...

Reply:
done